docs/api-reference/deckgl.md

```diff
@@ -119,5 +119,23 @@ object for the topmost picked layer at the coordinate
 are affected.
 - `event` - the original [MouseEvent](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent) object
 
+### Methods
+
+##### `queryObjects(boundingBox, layerId)`
```

Comment:
layerId => layerIds

Should we call this `fastQueryObjects` or similar to emphasize that it is imprecise?